torch/_inductor/fx_passes/freezing_patterns.py

```diff
@@ -118,3 +124,132 @@ def addmm_fuse_replacement_second(inp, w1, w2, w3, b1, b2, b3):
         extra_check=check_concat_weights,
         exclusive_arg_names=("w1", "w2", "w3", "b1", "b2", "b3"),
     )
+
+
+@functools.lru_cache(None)
+def binary_folding_init():
+    _conv_args = [Arg() for _ in range(9)]
+    _computation_ops = [aten.convolution.default]
+    _binary_ops = [aten.add.Tensor, aten.sub.Tensor, aten.mul.Tensor, aten.div.Tensor]
+    _computation_calls = [CallFunction(aten.convolution.default, *_conv_args)]
+
+    def _check_conv_and_broadcast_op(conv_node, other):
+        if not (isinstance(other, torch.fx.Node) and other.op != "get_attr"):
+            return False
+        if conv_node.args[1].op != "get_attr" or (
+            conv_node.args[2] is not None and conv_node.args[2].op != "get_attr"
+        ):
```

Comment:
I feel it is more readable to factor out a function, something like `is_constant(conv_node.args[1])`  to check whether some input is constant.

Reply:
yes, updated.